src/sentry/autofix/webhooks.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+from sentry import analytics
+from sentry.autofix.utils import get_autofix_state_from_pr_id
+from sentry.models.organization import Organization
+from sentry.utils import metrics
+
+AUTOFIX_GITHUB_APP_USER_ID = 157164994
+
+
+def get_webhook_analytics_fields(autofix_state: dict[str, Any]) -> dict[str, Any]:
+    autofix_request = autofix_state.get("request", {})
+    return {
+        "project_id": autofix_request.get("project_id"),
+        "group_id": autofix_request.get("issue", {}).get("id"),
+        "run_id": autofix_state.get("run_id"),
+    }
```

Comment:
Could you break this down so it's easier to test and more modular? Something like:
```suggestion
    webhook_analytics_fields = {
        "project_id": autofix_request.get("project_id", None),
    }
    
    issue = autofix_request.get("issue", None)
    webhook_analytics_fields["group_id"] = issue.get("id", None) if issue else None
    
    ...
    return webhook_analytics_fields
```